recipes/ignition-cmake/all/conanfile.py

```diff
@@ -28,6 +29,10 @@ def _configure_cmake(self):
             return self._cmake
         self._cmake = CMake(self)
         self._cmake.definitions["CMAKE_INSTALL_DATAROOTDIR"] = "lib"
+        self._cmake.definitions["BUILD_TESTING"] = True
+        self._cmake.definitions["-BUILDSYSTEM_TESTING"] = False
```

Comment:
```suggestion
```

Do not build test and examples on Conan Center. The idea is not validate a project by its unit tests, but only generated package, by consuming the artifacts on test package.